roofit/roofitcore/CMakeLists.txt

```diff
@@ -482,4 +482,9 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_G
   target_link_libraries(RooFitCore PRIVATE -lgcc_s -lgcc)
 endif()
 
+target_compile_options(RooFitCore PUBLIC -DROOFIT_NEW_BATCH_INTERFACE)
```

Comment:
If this is set unconditionally, then why have this flag at all? Should this flag be removed?

Reply:
It will be. We planed to mark the old interface as deprecated and then remove it after 2 versions.